x-pack/plugins/infra/public/components/asset_details/components/alerts_tooltip_content.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiText, EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { ALERTS_DOC_HREF } from '../../../common/alerts/constants';
+import { LinkToAlertsHomePage } from '../links/link_to_alerts_page';
+
+export const AlertsTooltipContent = React.memo(() => {
+  const onClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    e.stopPropagation();
+  };
+
+  return (
+    <EuiText size="xs" onClick={onClick}>
+      <FormattedMessage
+        id="xpack.infra.assetDetails.alerts.tooltip.alertsLabel"
+        defaultMessage="Showing alerts for this host. You can create and manage alerts in {alerts}"
```

Comment:
That's surprisingly tricky to do - because the wrapper for the link inside `LinkToAlertsHomePage` the It is not `inline` and it shows the dot on a new line: 
<img width="421" alt="image" src="https://github.com/elastic/kibana/assets/14139027/2348e01e-e099-4cbb-b92b-c3864eb805f3">

I tried to change the formatted message to inline one but the `div` comes from `RedirectAppLinks`, so I tied to  put a `css` prop to `RedirectAppLinks` and pass it down but the styles are predefined inside [redirect_app_links.styles.ts](https://github.com/elastic/kibana/blob/main/packages/shared-ux/link/redirect_app/impl/src/redirect_app_links.styles.ts) So as a workaround I will put the dot inside the link text (it will be blue because of the link styles but it will be there)